src/test/resources/org/codehaus/mojo/set/versionless-01/pom.xml

```diff
@@ -26,9 +26,12 @@
     <build>
         <plugins>
             <plugin>
+                <groupId>org.codehaus.mojo</groupId>
                 <artifactId>versions-maven-plugin</artifactId>
+                <goals>
+                    <goal>set</goal>
+                </goals>
                 <configuration>
-                    <updateBuildOutputTimestampPolicy>onchange</updateBuildOutputTimestampPolicy>
```

Comment:
why?

Reply:
not needed anymore since we're simply using the default value